### ISSUE TITLE
Custom "out of bounds" padding value for `jax.image.scale_and_translate`

### ISSUE THREAD

**Issue by josephrocca** [google/jax#7945]
Currently the behavior is to fill "out of bounds" pixels with `0`. This function would be more useful if this value could be cusomtized - like `cval` of `jax.scipy.ndimage.map_coordinates`. My particular use case is as part of an image augmentation pipeline.
